.github/workflows/openjdk_14_build.yml

```diff
@@ -0,0 +1,140 @@
+                  "blocks": [
+                      {
+                          "type": "section",
+                          "text": {
+                              "type": "mrkdwn",
+                              "text": "*SonarCloud Scan Failure*"
+                          },
+                          "fields": [
+                              {
+                                  "type": "mrkdwn",
+                                  "text": "*_Commit Author:_*"
+                              },
+                              {
+                                  "type": "mrkdwn",
+                                  "text": "*_Branch:_*"
+                              },
+                              {
+                                  "type": "mrkdwn",
+                                  "text": "${{github.actor}}"
+                              },
+                              {
+                                  "type": "mrkdwn",
+                                  "text": "${{github.ref_name}}"
+                              }
+                          ]
+                      },
+                      {
+                          "type": "section",
+                          "fields": [
+                              {
+                                  "type": "mrkdwn",
+                                  "text": "*_Github Actions Build Number:_*"
+                              },
+                              {
+                                  "type": "mrkdwn",
+                                  "text": "*_SonarCloud Analysis:_*"
+                              },
+                              {
+                                  "type": "mrkdwn",
+                                  "text": "<${{github.server_url}}/${{github.repository}}/actions/runs/${{github.run_id}}|${{github.run_number}}>"
+                              },
+                              {
+                                  "type": "mrkdwn",
+                                  "text": "<${{ steps.sonar_gate.outputs.sonar_dashboard_link }}|${{github.ref_name}}>"
+                              }
+                          ]
+                      },
+                      {
+                          "type": "section",
+                          "text": {
+                              "type": "mrkdwn",
+                              "text": "*_Git Commit:_*\n${{github.sha}}"
+                          }
+                      },
+                      {
+                          "type": "context",
+                          "elements": [
+                              {
+                                  "type": "mrkdwn",
+                                  "text": ":rotating_light: Please correct the issues identified by SonarCloud before merging!"
+                              }
+                          ]
+                      }
+                  ]
+            }
+        env:
+          SLACK_BOT_TOKEN: ${{ secrets.SLACK_ACCESS_TOKEN }}
```

Comment:
the token from circleci doesn't seem to be working, I don't think anyone is watching for these notifications though...

Reply:
I didn't even remember there was a slack option. TBH since I was the only one looking at CircleCI, it didn't really matter. WE might want to fix it for the future

Reply:
I’ll set this up properly without custom payload. Who would I talk to about getting a new slack token?

Reply:
done